src/tests/serde.rs

```diff
@@ -749,25 +742,25 @@ fn test_unsigned_helpers() {
 
 #[test]
 fn test_datetime_helpers() {
```

Comment:
This test function is getting pretty long - what do you think about breaking it up into smaller functions (maybe one for each helper)?